packages/core/src/pages/_app.tsx

```diff
@@ -1,17 +1,12 @@
-// FastStore UI's base styles
-import '../styles/global/index.scss'
-
-import '../customizations/src/themes/index.scss'
-
+import { UIProvider } from '@faststore/ui'
```

Comment:
@emersonlaurentino are you using a new plugin for indentation? I noticed that some files changed.

Reply:
No, the prettier that does this automatically.